src/js/viz2.js

```diff
@@ -9,6 +9,9 @@ var scores = {
   reverse: {}
 };
 
+var viz2_output = {};
```

Comment:
@chance-nelson this is what you want to pass to the backend after the submitPrimers function has been executed